src/views/playground/Playgrounds/PlaygroundUploadingFilesData.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { useState, useRef } from 'react'
+import { useUploadingFilesData } from '@/hooks'
+import styled from '@emotion/styled'
+import ImageCropDialog, { ImageCropDialogImperativeHandle } from '@/components/Dialogs/ImageCropDialog/ImageCropDialog'
```

Comment:
I think we should actually export this directly from `@/components`. That's carried over but could you clean that up?